tests/api/route-services/access/access.test.js

```diff
@@ -197,4 +207,14 @@ describe('tests for the projects service', () => {
     expect(getMock).toHaveBeenCalledTimes(2);
     expect(deleteMock).toHaveBeenCalledTimes(0);
   });
+
+  test('inviting a user works', async () => {
+    const grantRoleMock = jest.fn();
+    const as = new AccessService();
+    as.grantRole = grantRoleMock;
+    await as.inviteUser(fake.USER.email, fake.EXPERIMENT_ID, fake.PROJECT_ID, 'admin', { email: 'inviter@user.com' });
+
+    expect(grantRoleMock).toHaveBeenCalledTimes(1);
+    expect(grantRoleMock.mock.calls).toEqual([['00db34d7-5058-413b-b550-5f3fb4cdc5cc', 'experimentid11111111111111111111', 'aaaaaaaa-bbbb-3333-4444-999999999999', 'admin']]);
+  });
```

Comment:
add another test that fails when the user is not registered so it will fail when we add the new feature of inviting non-registered users